target/esp32/ll_cam.c

```diff
@@ -35,9 +35,7 @@ static inline int gpio_ll_get_level(gpio_dev_t *hw, int gpio_num)
 #include "cam_hal.h"
 
 #if (ESP_IDF_VERSION_MAJOR >= 5)
-#define GPIO_PIN_INTR_POSEDGE GPIO_INTR_POSEDGE
-#define GPIO_PIN_INTR_NEGEDGE GPIO_INTR_NEGEDGE
-#define gpio_matrix_in(a,b,c) gpio_iomux_in(a,b)
+#include "rom/gpio.h"
```

Comment:
This file will finally be removed in IDF 5.0, could you change this to `esp_rom_gpio.h`?
esp_rom_gpio.h exists since release/4.3.